cou/steps/execute.py

```diff
@@ -57,68 +57,86 @@ def normal(text: str) -> str:
         """
         return Style.RESET_ALL + text + Style.RESET_ALL
 
-    return normal(parameter + "Continue(") + bold("y") + normal("/") + bold("n") + normal("):")
+    return normal(parameter + "\nContinue (") + bold("y") + normal("/") + bold("n") + normal("): ")
 
 
-async def _run_step(step: BaseStep, interactive: bool) -> None:
+async def _run_step(step: BaseStep, interactive: bool, overwrite_progress: bool = False) -> None:
```

Comment:
What do you think about isolating the logic to decide if should overwrite_progress or not in a separate function and not have as parameter of `_run_step` and `apply_step`?

I think it will be cleaner and we can easily maintain.

Reply:
I don't see a way of easily doing so because this logic requires some recursive check (all sub-steps of ApplicationUpgradePlan's sub-steps would need to set it to `True`). Plus the actual logic is a one-line code with clear comment, so I think it is ok to keep the implementation as-is.